sagemcom_api/client.py

```diff
@@ -328,15 +334,36 @@ async def set_value_by_xpath(
         }
 
         response = await self.__api_request_async([actions], False)
-        print(response)
 
         return response
 
     async def get_device_info(self) -> DeviceInfo:
         """Retrieve information about Sagemcom F@st device."""
-        data = await self.get_value_by_xpath("Device/DeviceInfo")
-
-        return DeviceInfo(**data.get("device_info"))
+        try:
+            data = await self.get_value_by_xpath("Device/DeviceInfo")
+            return DeviceInfo(**data.get("device_info"))
+        except UnknownPathException:
+            device_info = DeviceInfo()
```

Comment:
In a follow up PR, it would be good to try catch this as well, since this could also throw an `UnknownPathException`.

Reply:
What do you propose to do if this also throws an exception?
I do not see any way to recover from that, so why not just let the exception propagate?

Reply:
You are right, we need to catch this in the Home Assistant integration :-)